website/package.json

```diff
@@ -15,8 +15,8 @@
     "test": "tests"
   },
   "scripts": {
-    "prebuild": "node lib/changelog/generate-changelog-markdown-files.mjs",
-    "build": "ember build --environment=production",
+    "generate-changelog-markdown-files": "node lib/changelog/generate-changelog-markdown-files.mjs",
+    "build": "yarn generate-changelog-markdown-files && ember build --environment=production",
```

Comment:
I'm curious the thinking of including this in the build as well: is this a workaround for the fact that `prebuild` doesn't actually do anything? If we are updating this as part of the `version` step I _think_ this would not actually ever trigger new changes. Am I thinking about that correctly?

Reply:
it's just in case, until we find out if the `yarn version` works, then we can remove it